src/components/export/export.js

```diff
@@ -159,8 +160,11 @@ export default class Export extends Component {
         // Multicall args to display for copy/paste
         let multicallArgs = "";
         if (this.showArgs) {
-            // Return error message if cards still have errors. Faulty JSON breaks toBase64.
-            if (errors.hasErrors.isBad) { multicallArgs = errors.hasErrors.message; }
+            // Return error message if a card has JSON errors. Faulty JSON breaks toBase64.
+            const hasJsonErrors = errors.hasErrors.isBad && !!allErrors.find(err => err.message === errorMsg.ERR_INVALID_ARGS);
```

Comment:
Little risky, but i hope my upcoming args refactoring can turn this into something more sophisticated. Will do for now 👍🏽 
Btw, array.some does what u need here and will also return a boolean.

Reply:
good point about `array.some()`, thx!